airflow/models/dag.py

```diff
@@ -2781,6 +2801,20 @@ def validate_schedule_and_params(self):
                     "DAG Schedule must be None, if there are any required params without default values"
                 )
 
+    def validate_owner_links(self) -> Dict[str, str]:
+        """Parses a given link, and verifies if it's a valid URL, or a 'mailto' link"""
+        wrong_links = {}
+        for owner, link in self.owner_links.items():
+            result = urlparse(link)
+            if link.startswith('mailto:'):
```

Comment:
```suggestion
            if result.scheme == 'mailto':
```

Since we already parsed the link, probably safer to rely on the result